docs/checkpoints.md

```diff
@@ -0,0 +1,70 @@
+<!--
+Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Checkpointing in Model Analyzer
+
+The Model Analyzer's `profile` subcommand writes the measurements it has
+collected to disk periodically as pickle files called checkpoints. These are
+located within the specified checkpoint directory (See [Config
+Defaults](./config.md#Config-Defaults) section for default location). The
+`analyze` and `report` subcommands read the measurements from a checkpoint file
+and process them to create their outputs.
+
+## When is Checkpointing Done?
+
+Model Analyzer saves a checkpoint in multiple circumstances:
+1. First and foremost, Model Analyzer will save a checkpoint after all the perf
+   analyzer runs for a given model are complete.
+2. Second, the user can initiate an early exit from profiling using `CTRL-C
+   (SIGINT)`. This will wait for the current perf analyzer run to finish and
+   then save a checkpoint before exiting.
+3. If the user needs to exit immediately, they send the `SIGINT` 3 times. In
+   this case, Model Analyzer will save a checkpoint and exit immediately.
+
+## Checkpoint Naming Scheme
+
+Suppose a `profile` run finishes.
+
+```
+$ model-analyzer profile -m example_model_repo --profile-models example_model_1,example_model_2
+2021-05-13 19:57:05.87 INFO[entrypoint.py:98] Starting a local Triton Server...
+2021-05-13 19:57:05.92 INFO[server_local.py:64] Triton Server started.
+2021-05-13 19:57:09.234 INFO[server_local.py:81] Triton Server stopped.
+2021-05-13 19:57:09.235 INFO[analyzer_state_manager.py:118] No checkpoint file found, starting a fresh run.
+.
+.
+.
+2021-05-13 19:58:01.625 INFO[analyzer.py:110] Finished profiling. Obtained measurements for models: ['example_model_1', 'example_model_2']
+
+```
+
+If we check the contents default checkpoint directory, we should find that there
```

Comment:
Do not use I, we, us, they, etc. Just say, "This will result in two checkpoints in the default checkpoint directory"

Reply:
Do not write in a passive tone either. Avoid "to be" verbs and languages. In the above, "should find" is passive. Sounds like there are cases where we "should not" find. In tech writing, be absolute and active. "We find", "There are", "this is", etc.